mirrormanager2/app.py

```diff
@@ -380,7 +383,13 @@ def host_new(site_id):
             APP.logger.debug('Could not create the new host')
             APP.logger.exception(err)
 
+        message = dict(
+            status=status,
```

Comment:
@pypingou @ralphbean , am so sorry. dont know how i missed that. Will be more careful next time.